specification/src/main/asciidoc/platform/TransactionManagement.adoc

```diff
@@ -176,10 +175,10 @@ between the use of transactional resources and threads. To ensure
 correct operation, web components should obey the following guidelines,
 and the web container must support at least these usages.
 
-* JTA transactions should be started and
+* Jakarta Transactions should be started and
```

Comment:
"Jakarta Transactions transactions" I think

Reply:
Or just "Transactions"?  Or "Transactions defined by the Jakarta Transactions specification"?

"Jakarta Transactions transactions" is just too clumsy...

Reply:
Adopted to referring "Transactions defined by the Jakarta Transactions specification".